experimental/dds/sequence-deprecated/api-report/sequence-deprecated.api.md

```diff
@@ -188,7 +188,7 @@ export class SparseMatrixFactory implements IChannelFactory {
 }
 
 // @public @deprecated (undocumented)
-export type SparseMatrixItem = Serializable;
+export type SparseMatrixItem = any;
```

Comment:
Not for this PR, but question for other reviewers: can we delete this and other deprecated aspects of Matrix? There's a fair amount of old cruft here that's been deprecated for a long time.

Reply:
There is fear that this will break ancient documents created with pre-release versions of Loop.

(I'm not sure this is true, but research will be required.)